src/components/link/style.scss

```diff
@@ -0,0 +1,19 @@
+.link-text{
```

Comment:
1. Use the sass syntax to be more specific.
2. Try reusing the classes you already have.
3. Also try being DRY, there is no need for that much CSS.